examples/common_nestjs_remix/apps/web/react-router.config.ts

```diff
@@ -0,0 +1,5 @@
+import type { Config } from "@react-router/dev/config";
+
+export default {
+  ssr: false,
+} satisfies Config;
```

Comment:
let's add this to vite.config instead of new config file - one less file to worry about and we can have the same